### PR TITLE
HTML Reporting in Unit testing

### PR DESCRIPTION
**Pull request by dmarvp**
- Fixes a typo in the e2e html reporting section
- Adds the same html reporting changes in the unit testing guides for alexa and google
- Adds screens to the code coverage sections too
- Moves the HTML Reporting, Code Coverage and Continous Integration sections up by one level

### CODE REVIEW

**Review by jperata — APPROVED** [review]

File: unit-testing/guide-google.md [hunk 1]
```diff
@@ -671,22 +671,43 @@ Remember you can also use the override properties when executing the bst test co
...
 
-### HTML Reporting
-The results of your tests are automatically formatted into a nice HTML report, courtesy of jest-stare.
+An HTML report is also viewable under `./test_output/coverage/lcov-report/index.html`.
 
-It is viewable under `./test_output/results/index.html`.
+![files](https://user-images.githubusercontent.com/6411740/69264437-527a4980-0b95-11ea-8066-c5506c2e23f6.png)
 
-It provides a nice summary of the results of your tests, with charts. You can also drill down into the detailed test results.
+Clicking on a file will reveal the lines that were covered (hihghlighted in green) and the ones that were not (highlighted in red).
```
Comment (jperata) [inline unit-testing/guide-google.md]: typo hihghlighted

File: unit-testing/guide-google.md [hunk 2]
```diff
@@ -671,22 +671,43 @@ Remember you can also use the override properties when executing the bst test co
...
+
+You can filter the tests by result with the toggles at the top.
+
+![result toggles](https://user-images.githubusercontent.com/6411740/69192392-7769b080-0af2-11ea-99e5-72a15040be5e.png)
+
+By scrolling down or clicking on any of the tests of the summary, you can go into the detailed test results.
+
+![detailed results](https://user-images.githubusercontent.com/6411740/69192709-24dcc400-0af3-11ea-941b-3b26fed8aaa1.png)
+
+You can also customize the title at the start of the report and the title of the report window by setting the following environment variables repectively:
```
Comment (jperata) [inline unit-testing/guide-google.md]: typo repectively

File: unit-testing/guide.md [hunk 3]
```diff
@@ -657,22 +657,43 @@ configuration:
...
+![code coverage](https://user-images.githubusercontent.com/6411740/69264113-acc6da80-0b94-11ea-96af-464e3acf8f39.png)
+
 An HTML report is also viewable under `./test_output/coverage/lcov-report/index.html`.
-TESTING_CONFIG_DIR is the directory where your `testing.json` file is located.
 
-### HTML Reporting
-The results of your tests are automatically formatted into a nice HTML report, courtesy of jest-stare.
+![files](https://user-images.githubusercontent.com/6411740/69264437-527a4980-0b95-11ea-8066-c5506c2e23f6.png)
+
+Clicking on a file will reveal the lines that were covered (hihghlighted in green) and the ones that were not (highlighted in red).
```
Comment (jperata) [inline unit-testing/guide.md]: typo hihghlighted

File: unit-testing/guide.md [hunk 4]
```diff
@@ -657,22 +657,43 @@ configuration:
...
-It provides a nice summary of the results of your tests, with charts. You can also drill down into the detailed test results.
+You can filter the tests by result with the toggles at the top.
 
+![result toggles](https://user-images.githubusercontent.com/6411740/69192392-7769b080-0af2-11ea-99e5-72a15040be5e.png)
+
+By scrolling down or clicking on any of the tests of the summary, you can go into the detailed test results.
+
+![detailed results](https://user-images.githubusercontent.com/6411740/69192709-24dcc400-0af3-11ea-941b-3b26fed8aaa1.png)
+
+You can also customize the title at the start of the report and the title of the report window by setting the following environment variables repectively:
```
Comment (jperata) [inline unit-testing/guide.md]: typo repectively

**Review by jperata — CHANGES_REQUESTED** [review]

**Review by jkelvie — APPROVED** [review]

**Pull request merged**